src/colp/components/usetype_version_comparison_report.py

```diff
@@ -0,0 +1,76 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import pdb
+
+
+class UsetypeVersionComparisonReport:
+    def __init__(self, usetype_changes, version):
+        self.usetype_changes = usetype_changes
+        self.version = version
+
+    def __call__(self):
+        st.subheader("Usetype Version Comparison")
+
+        if self.usetype_changes is None:
+            st.info("There is no usetype version comparison report for this branch.")
+            return
+
+        self.current_usetype_changes = self.filter_by_version(
+            self.usetype_changes, self.version
+        )
+
+        st.markdown(
+            f"""
+            This report compares the usetype distribution between the previous and selected versions.
```

Comment:
We can give more context here as to why we do this check. I can add this at another point though but the idea here is that if we see even minor swings in the number of these "stable" use types than it warrants a proper investigation. e.g. Rarely does the city build or demolish firehouses and I believe at one point we lost two firehouses from version to version which had to be investigated by lynn

Reply:
Have you opened an issue for this?